scripts/digest_inputs

```diff
@@ -54,18 +67,40 @@ add_vars TAGS REDHAT_TAG DEV_TAGS WORKDIR DOCKERFILE
 
 # Determine repo name values.
 REPO_NAME_MINUS_ENTERPRISE="${REPO_NAME%%-enterprise}"
+
 # Assume this is not enterprise.
 ENTERPRISE_DETECTED=false
-ENT_META=""
+
+#
+## Version mangling.
+#
+# Catch and auto-correct a few common mistakes with the version input.
+# Emit loud warnings when action is taken based on these rules.
+
 # This checks if removing '-enterprise' suffix actually did anything.
 # If it did, then we've detected a likely enterprise repo.
 [ "$REPO_NAME" != "$REPO_NAME_MINUS_ENTERPRISE" ] && {
   ENTERPRISE_DETECTED=true
-  if ! [[ "$VERSION" =~ .*\+.* ]]; then
-	VERSION="$VERSION+ent"
+  # If the version doesn't contain a '+' or '-ent' then add the '+ent' suffix.
```

Comment:
I don't know if we should append a `+ent` here to all enterprise products since we don't have the same logic to append one for the binaries or linux packages. So to make it consistent we should use the version directly as is like all the other artifacts. Thoughts?

Reply:
Ok, my understanding was that we were sometimes asking teams not to include the `+ent` part of the version string, and I couldn't remember why? It would certainly be simpler if we told all teams they need to use the complete version string, be it in any format, e.g. `X.Y.Z` `X.Y.Z.N` `X.Y.Z-prerelease` `X.Y.Z+ent` `X.Y.Z+ent.fips124` or `X.Y.Z-prerelease+ent.fips124` etc..

Reply:
I would rather not change this logic in this PR because it was added a long time ago, so some teams may be dependent on it. We should remove the logic in a follow-up and release a major version bump I think.

Reply:
That sounds good to me, I think it would require some discussion with the RDX folks on exactly what teams are doing too so lets keep it as is for now.